src/server/http/validation_error_handler.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// Used to replicate Hapi v16 and below's validation responses
+export function defaultValidationErrorHandler(request, h, err) {
+  throw err;
```

Comment:
Hmm, what's the default behavior in Hapi 17 that is different from 15? Is this possibly necessary because we are returning errors rather than throwing them?

Reply:
No, Hapi stopped exposing the actual details of validation errors. This is the community recommended way of exposing these errors the same way as before.

Reply:
Ah, I missed that this is specifically for the `validate:` route config. 👍

Reply:
I guess I didn't answer your original question: it doesn't matter whether your throw or return the Boom object.